packages/integration-karma/test/rendering/elements-are-not-recycled/index.spec.js

```diff
@@ -0,0 +1,44 @@
+import { createElement } from 'lwc';
+import Container from 'x/container';
+
+it('should not have have an error re-rendering slotted content', function() {
```

Comment:
Where do we expect to see the error if elements were recycled? We should use a `expect().not.toThrow()` there.

Reply:
i dont think so, just by rendering the component we know it does not throw right?